contracts/ethregistrar/BaseRegistrar.sol

```diff
@@ -1,51 +1,32 @@
 pragma solidity ^0.8.4;
```

Comment:
Do we actually need this file anymore? Can't we move the constant and events into `IBaseRegistrar`, and the mapping into `BaseRegistrarImplementation`?

Reply:
I guess so, but since BaseRegistrarImplementation is already deployed and likely never to be redeployed - is it worth making these changes? If you think so I can change it